app.go

```diff
@@ -693,7 +749,11 @@ func (app *App) executeInvokes() error {
 		}
 
 		if err != nil {
-			app.logger.Printf("fx.Invoke(%v) called from:\n%+vFailed: %v", fname, i.Stack, err)
+			fxlog.Error(
+				"fx.Invoke failed",
+				fxlog.F("calledFrom", fname),
```

Comment:
I know we already renamed this but it may be better to align it with the field name we already use for functions.

```suggestion
				fxlog.F("function", fname),
```